src/App.tsx

```diff
@@ -1,42 +1,34 @@
-import React from 'react';
+import { useState, FC } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { tabs } from './tabData';
+import { Tab } from './types';
 
-export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-];
+export const App: FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+  const isActive
+    = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const selectTab = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      setSelectedTabId(tab.id);
+    }
```

Comment:
I like to avoid nesting as much as possible ;)
```suggestion
    if (tab.id === selectedTabId) return;
      
    setSelectedTabId(tab.id);
```